app/Enum/WebhookEvent.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
to all other files i created? I'm not sure if thats what you meant, because right now `declare(strict_types=1);` is only used in the enum files

Reply:
Yes, this would be great. But it is non blocking for me.
Strict types are nice 2 have at the beginning in a file, so we are forced to use explizit types in every file and improve our type safety.

Reply:
i think it would be better to do this in an extra PR for the whole project when this PR is merged